pkg/datarepair/checker/checker.go

```diff
@@ -25,6 +27,7 @@ type Checker interface {
 
 // Checker contains the information needed to do checks for missing pieces
 type checker struct {
+	statdb      *statdb.Server
```

Comment:
This is `statdb.StatDB` now. It is no longer a grpc server

Reply:
done